js/pages/characterizations/components/feature-analyses/feature-analysis-view-edit.js

```diff
@@ -202,6 +202,7 @@ define([
         }
 
         setupAnalysisData({ id = 0, name = '', descr = '', domain = '', type = '', design= '', conceptSets = [], statType = 'PREVALENCE' }) {
+            domain = !!domain ? domain : !!this.domains()[0] && !!this.domains()[0].value ? this.domains()[0].value : '';
```

Comment:
This is not readable. Cannot we introduce a couple of variables with seld-descriptive names?

Reply:
Fixed